pip-requirements.txt

```diff
@@ -0,0 +1,2 @@
+mock>=2.0.0
+coveralls
```

Comment:
pip requirements missing `nose`. You get away with this because both `mock` and `nose` come pre-installed, but you should never the less have it specified in your requirements for other developers.